docs/registries.1.md

```diff
@@ -69,6 +69,18 @@ default input file:
 # registries -i /tmp/new_file
 ```
 
+# MIGRATION FROM YAML
+The initial registries.conf files where based on YAML structure.  This registries too
+still supports reading YAML.  However, you should preferably migrate the YAML to TOML.
```

Comment:
Is it really still supported  Unless I'm mis-reading the code, it looks like you're forcing a migration to TOML from YAML, although you're helping with the translation/migration.  Perhaps:

This version of registries now requires TOML configuration files, but does provide a migration function for YAML configuration files to TOML configuration files.  To migrate a YAML configuration file to the TOML format, make a backup copy ...

Or some such.

Reply:
Nope, you're supporting both, I didn't read down far enough into the code.

Reply:
Ok, my $.02.  I'm perfectly fine with supporting only TOML, in fact I'd prefer supporting just it instead of both TOML and YAML.  If it were my decision, I'd go with stating we're moving to TOML, but we've provided a YAML migration tool to help the migration for you.  Something similar to one of my comments above.

But I think that's a call for Dan and perhaps others. 

If we're doing both after all, there are a couple or *.md files that you changed from YAML to TOML.  We might want to make those "YAML and TOML".

The code looks fine to me fwiw.